Description: Remove initial value for OL "start" attribute.
 Causes validate-html to fail and also causes net/html result to differ.
Author: Doug Torrance <dtorrance@debian.org>
Bug: https://github.com/Macaulay2/M2/pull/2760/files#r1133175427
Last-Update: 2023-03-11

--- a/M2/Macaulay2/m2/hypertext.m2
+++ b/M2/Macaulay2/m2/hypertext.m2
@@ -355,7 +355,7 @@
 addAttribute(TH,     htmlAttr | {"colspan", "headers", "rowspan"})
 addAttribute(IMG,    htmlAttr | {"alt", "src", "srcset", "width", "height",
 	"sizes", "crossorigin", "longdesc", "referrerpolicy", "ismap", "usemap"})
-addAttribute(OL, htmlAttr | {"start"=>"0", "reversed", "type"})
+addAttribute(OL, htmlAttr | {"start", "reversed", "type"})
 buttonAttr = htmlAttr | {"autofocus","disabled",
     "form","formaction","formenctype","formmethod","formnovalidate","formtarget",
     "name", "type", "value"}
